test/e2e/deprecated_api_test.go

```diff
@@ -0,0 +1,61 @@
+package e2e
+
+import (
+	"context"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	test "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+	monitoringclient "github.com/prometheus-operator/prometheus-operator/pkg/client/versioned"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/discovery"
+)
+
+func TestDeprecatedAPIInUse(t *testing.T) {
```

Comment:
good test.  It will break every kube bump.  I find this acceptable since it won't block the kube bump.

Reply:
:+1: @sanchezl ignore my ask for a test that breaks elsewhere. This is it I guess.